mesonbuild/wrap/wrap.py

```diff
@@ -67,93 +67,103 @@ def open_wrapdburl(urlstring):
         urlstring = 'http' + urlstring[5:]
     return urllib.request.urlopen(urlstring, timeout=req_timeout)
 
+class WrapException(MesonException):
+    pass
+
+class WrapNotFoundException(WrapException):
+    pass
 
 class PackageDefinition:
     def __init__(self, fname):
-        self.values = {}
-        with open(fname) as ifile:
-            first = ifile.readline().strip()
-
-            if first == '[wrap-file]':
-                self.type = 'file'
-            elif first == '[wrap-git]':
-                self.type = 'git'
-            elif first == '[wrap-hg]':
-                self.type = 'hg'
-            elif first == '[wrap-svn]':
-                self.type = 'svn'
-            else:
-                raise RuntimeError('Invalid format of package file')
-            for line in ifile:
-                line = line.strip()
-                if line == '':
-                    continue
-                (k, v) = line.split('=', 1)
-                k = k.strip()
-                v = v.strip()
-                self.values[k] = v
+        self.basename = os.path.basename(fname)
+        try:
+            self.config = configparser.ConfigParser()
```

Comment:
`interpolation=None`?
Or do we want to allow interpolations in wrap files?

Reply:
Better disable it for now, we can always enable later if needed. Added a patch for that.